app/models/parcel.rb

```diff
@@ -40,4 +43,10 @@ def generate_parcel_number
       break parcel_number unless Parcel.exists?(parcel_number: parcel_number)
     end
   end
+
+  def create_order_created_operation
+    operation = Operation.new(parcel: self, kind: 'order_created')
+    operation.user = self.sender if self.sender
```

Comment:
Po co if? Jeśli `sender` będzie nilem, to `user` też nilem pozostanie.
Btw - prefix `self.` jest zbędny przy wołaniu metod instancji - natomiast rozumiem, jeśli chciałbyś z nich korzystać, może to być traktowane jako kwestia osobistego stylu.